### ISSUE TITLE
should focusable children of items marked with aria-hidden="true" not be focusable

### ISSUE THREAD

**Issue by RichCaloggero** [nvaccess/nvda#8848]
## Feature request / enhancement

Currently, focusable children of elements marked with `aria-hidden="true"` still gain focus via tab.  Seems to me that NVDA should prevent this if possible. The browser cannot do this as it knows nothing about whether a screen reader is running.

**Comment by Adriani90** [comment]
@RichCaloggero could you please upload a test case?

**Comment by Adriani90** [comment]
@RichCaloggero this should only be ignored if the element is not visible on the screen, i.e. visibility:hidden or display:none applies. If the element is focusable, and visible, then even with aria hidden it is expected to gain focus also by screen readers. This is conform with Aria spec, see this link:
https://www.w3.org/WAI/PF/aria-1.1/states_and_properties#aria-hidden
For more details and if you need more clarification, the dicussion can be continued on gitter at
https://gitter.im/nvaccess/NVDA
I am closing this for now.
